react-components/src/library/components/assign-to-class/style.scss

```diff
@@ -164,6 +164,39 @@
   }
 }
 
+.confirmDialogPortal {
+  left: 0;
+  position: absolute;
+  top: 0;
+}
+
+.confirmDialogOverlay {
+  background-color: rgba(0,0,0,0.3);
+  height: 100%;
+  position: fixed;
+  width: 100%;
+  z-index: 101;
+}
+.confirmDialog {
+  background: #fff;
+  border: none;
+  border-radius: 0;
+  bottom: auto;
+  left: 50%;
+  margin-right: -50%;
+  outline: none;
+  padding: 20px;
+  position: absolute;
+  right: auto;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 450px;
+
+  button {
+    margin: 0 auto;
+  }
+}
+
```

Comment:
@scytacki: I had some trouble getting custom CSS applied to the new modal component and ended up adding it here. It works OK, but I suspect there's a better way to do it so the modal styles are kept consistent if/when React Modal is used outside of assigning resources to classes.

Should I spend some time on that now? I don't want to delay the next update any longer if this is the only thing holding it up.